<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #000000</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,20">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="login_form_stacked_widget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>407</width>
       <height>298</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #262626</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="stackedWidgetPage1">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>140</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="password_edit">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>140</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #FFFFFF</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="username_edit">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>110</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #FFFFFF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="login_button">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>180</y>
         <width>75</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #262626;
color:white;
border:none;
}
QPushButton:hover{
background-color: #323232;
}
QPushButton:disabled{
background-color: #FFFFFF;
color: #262626;
}</string>
       </property>
       <property name="text">
        <string>Log in</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>110</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="left_panel_icon_label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>10</y>
         <width>211</width>
         <height>91</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>LOGO_login.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="server_setting_button">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>180</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #262626;
color:white;
border:none;
}
QPushButton:hover{
background-color: #323232;
}
QPushButton:disabled{
background-color: #FFFFFF;
color: #262626;
}</string>
       </property>
       <property name="text">
        <string>Server setting</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page">
      <widget class="QLineEdit" name="sftp_host_edit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>130</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #FFFFFF</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="mysql_host_edit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>100</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #FFFFFF</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>MySQL host</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>SFTP host</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>MySQL port</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="mysql_port_edit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>160</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #FFFFFF</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="sftp_port_edit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>190</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #FFFFFF</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>SFTP port</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="server_setting_save_button">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>250</y>
         <width>75</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #262626;
color:white;
border:none;
}
QPushButton:hover{
background-color: #323232;
}
QPushButton:disabled{
background-color: #FFFFFF;
color: #262626;
}</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QPushButton" name="server_Setting_cancel_button">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>250</y>
         <width>75</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #262626;
color:white;
border:none;
}
QPushButton:hover{
background-color: #323232;
}
QPushButton:disabled{
background-color: #FFFFFF;
color: #262626;
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QLabel" name="left_panel_icon_label_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>10</y>
         <width>211</width>
         <height>91</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../.designer/backup/그림1.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <zorder>server_setting_save_button</zorder>
      <zorder>server_Setting_cancel_button</zorder>
      <zorder>left_panel_icon_label_2</zorder>
      <zorder>label_5</zorder>
      <zorder>mysql_host_edit</zorder>
      <zorder>label_6</zorder>
      <zorder>sftp_host_edit</zorder>
      <zorder>label_4</zorder>
      <zorder>mysql_port_edit</zorder>
      <zorder>sftp_port_edit</zorder>
      <zorder>label_3</zorder>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: #FFFFFF</string>
     </property>
     <widget class="QPushButton" name="minimize_button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #FFFF;
color:black;
border:none;
}
QPushButton:hover{
background-color: #DCDCDC;
}
QPushButton:disabled{
background-color: #FFFFFF;
color: #262626;
}</string>
      </property>
      <property name="text">
       <string>_</string>
      </property>
     </widget>
     <widget class="QPushButton" name="close_button">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: #FFFF;
color:black;
border:none;
}
QPushButton:hover{
background-color: #FF0000;
}
QPushButton:disabled{
background-color: #FFFFFF;
color: #262626;
}</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mysql_host_edit</tabstop>
  <tabstop>sftp_host_edit</tabstop>
  <tabstop>mysql_port_edit</tabstop>
  <tabstop>sftp_port_edit</tabstop>
  <tabstop>server_Setting_cancel_button</tabstop>
  <tabstop>server_setting_save_button</tabstop>
  <tabstop>username_edit</tabstop>
  <tabstop>password_edit</tabstop>
  <tabstop>server_setting_button</tabstop>
  <tabstop>login_button</tabstop>
  <tabstop>minimize_button</tabstop>
  <tabstop>close_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minimize_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>server_setting_save_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>server_setting_save_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>server_Setting_cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>server_setting_cancel_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>login_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>login_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>server_setting_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>server_setting_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>password_edit</sender>
   <signal>returnPressed()</signal>
   <receiver>Form</receiver>
   <slot>login_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>username_edit</sender>
   <signal>returnPressed()</signal>
   <receiver>Form</receiver>
   <slot>login_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>login_button_clicked()</slot>
  <slot>server_setting_save_button_clicked()</slot>
  <slot>server_setting_cancel_button_clicked()</slot>
  <slot>server_setting_button_clicked()</slot>
 </slots>
</ui>
